magento/magento2
# Fix filter placeholders for ui-select filter (frontend implementation)

### Description (*)


### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/adobe-stock-integration#1780: "Active filters" shows the "Asset:" filter which is empty if switch to the Default View again 

### Manual testing scenarios (*)
1. Verify asset filter placeholders in grids -> product, pages, blocks, categories
2. Verify asset filter placeholder with saved grid views in all listed grid (products,pages,blocks,categories)
3. Verify asset filter placeholders from saved in bookmarks filters in listed grids products,pages,blocks,categories)
4. Verify Used in filters place holders in standalone and slide panel media gallery with saved in bookmarks and saved in different grid views.
5. Verify url-filter applier by link from image view details thats used in  (products,pages,blocks,categories)

### Questions or comments


### Contribution checklist (*)
